client/src/page/Board.css

```diff
@@ -0,0 +1,33 @@
+.create-school-form{
+    --width: 50%;
+    --height: 380px;
+    --padding:40px;
+    position: absolute;
+    left:calc((100% - var(--width) - var(--padding))/2);
+    bottom:calc((100% - var(--height) )/2);;
+    display:flex;
+    flex-direction: column;
+    gap:35px;
+    width:var(--width);
+    height: var(--height);
+    align-items: center;
+
+    background-color: lightcyan;
```

Comment:
Beautiful CSS! Maybe after this sprint we can change the colors to be consistent across the entire app